Vagrantfile

```diff
@@ -77,6 +77,17 @@ def mount_folders(config, mount_options)
 end
 
 Vagrant.configure(2) do |config|
+  config.vm.boot_timeout=600
+  config.vm.provider :virtualbox do |v|
```

Comment:
We already have a `config.vm.provider` block and I'd suggest merging the two to have just one block.

Reply:
Yes I just saw that and I will merge it. Thanks